content/en/docs/azure/deploy/install-kubeflow.md

```diff
@@ -9,12 +9,12 @@ deploy Kubeflow on Azure.
 
 ## Prerequisites
 
--   Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux)
--   Install and configure the [Azure Command Line Interface (Az)](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
-	-  Log in with ```az login```
--   (Optional) Install Docker
-	-   For Windows and WSL: [Guide](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly)
-	-   For other OS: [Docker Desktop](https://hub.docker.com/?overlay=onboarding)
+- Install [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux)
+- Install and configure the [Azure Command Line Interface (Az)](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+  - Log in with ```az login```
+- (Optional) Install Docker
+  - For Windows and WSL: [Guide](https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly)
+  - For other OS: [Docker Desktop](https://hub.docker.com/?overlay=onboarding)
```

Comment:
Line 17: I think this Docker Hub link would take the user straight to a login page, which is not awesome in terms of UX, especially if they don't have an account yet. Maybe, a page from https://docs.docker.com/docker-hub/ would suffice?

Reply:
fixed